packages/scxa-faceted-search-results-v5/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "scxa-faceted-search-results-v5",
+  "version": "5.0.0",
+  "description": "Faceted experiments list for Single Cell Expression Atlas gene search results",
+  "keywords": [
+    "react",
+    "ebi",
+    "species",
+    "bioinformatics",
+    "facets",
+    "search"
+  ],
+  "author": "Karoly Erdos <karoly@ebi.ac.uk>",
+  "homepage": "https://github.com/ebi-gene-expression-group/atlas-components#readme",
+  "license": "Apache-2.0",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ebi-gene-expression-group/atlas-components.git"
+  },
+  "dependencies": {
+    "@ebi-gene-expression-group/atlas-react-fetch-loader": "*",
+    "format-number": "^3.0.0",
+    "lodash": "^4.17.20",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-select": "^3.1.0",
+    "react-tooltip": "^4.2.9",
+    "styled-components": "^5.1.1",
+    "urijs": "^1.19.2"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.11.6",
+    "@babel/core": "^7.11.6",
+    "@babel/polyfill": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@cypress/code-coverage": "^3.10.6",
+    "@ebi-gene-expression-group/eslint-config": "^3.5.1",
+    "@ebi-gene-expression-group/react-ebi-species": "^2.6.2",
+    "@testing-library/cypress": "^9.0.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.1.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "coveralls": "^3.1.0",
+    "cypress": "^12.16.0",
+    "eslint": "^8.17.0",
+    "fetch-mock": "^9.10.7",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^26.4.2",
+    "node-fetch": "^2.6.0",
+    "random-words": "1.2.1",
+    "react-test-renderer": "^16.13.1",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
```

Comment:
This stub script can go away, or be replaced with something more useful. If you want to use this, and want some inspiration, have a look at [our template package](https://github.com/ebi-gene-expression-group/atlas-package/blob/master/package.json#L11).